includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -629,7 +661,7 @@ public function sanitize_settings( $settings ) {
 		}
 
 		if ( isset( $settings['generate_title_prompt'] ) && ! empty( $settings['generate_title_prompt'] ) ) {
-			$new_settings['generate_title_prompt'] = sanitize_textarea_field( $settings['generate_title_prompt'] );
+			$new_settings['generate_title_prompt'] = $this->sanitize_prompts( $settings['generate_title_prompt'] );
 		} else {
 			$new_settings['generate_title_prompt'] = '';
```

Comment:
Should this be changed to match how the others work? Where we check if the value `is_array` and if not, we set the default to an empty array?

Reply:
@dkotter prompt value is a two-level array, so we should validate/sanitize it differently.